src/codeu/chat/common/BasicView.java

```diff
@@ -32,6 +32,7 @@
 
 public interface BasicView {
 
+  ServerInfo getInfo();
```

Comment:
Document the purpose of this method. In interfaces, especially since there is no method body to speak for itself, it's important to understand the motivation / usage of a method, especially since consumers of this interface may not have visibility into the implementation code.